cartographer_grpc/framework/rpc.cc

```diff
@@ -119,14 +124,13 @@ void Rpc::RequestStreamingReadIfNeeded() {
           new RpcEvent{Event::READ, weak_ptr_factory_(this), true});
       break;
     case ::grpc::internal::RpcMethod::NORMAL_RPC:
-      // For NORMAL_RPC we don't have to do anything here, since gRPC
-      // automatically issues a READ request and places the request into the
-      // 'Message' we provided to 'RequestAsyncUnary' above.
+    case ::grpc::internal::RpcMethod::SERVER_STREAMING:
+      // For NORMAL_RPC and SERVER_STREAMING we don't have to do anything here,
```

Comment:
replace "don't have to do anything" by "don't need to queue an event" or something similar

Reply:
Done.